app/src/main/res/layout/fragment_swipeable_pets.xml

```diff
@@ -1,7 +1,8 @@
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:clipChildren="false"
```

Comment:
this one little line took so long to figure out! This tells the layout to never clip our swipe card.